.github/workflows/pipeline.yml

```diff
@@ -21,7 +21,9 @@ jobs:
         pip install --upgrade pip
         pip install "tox<4" "tox-gh-actions<3" "setuptools<58" "coveralls<4"
     - name: Test with tox
-      run: tox
+      run: |
+        mkdir -p .mypy_cache
```

Comment:
Do we explicitly need to create this? Don't think this actually does something as we are running mypy once?